Aztec/Classes/Converters/NSAttributedStringToNodes.swift

```diff
@@ -395,14 +435,42 @@ private extension NSAttributedStringToNodes {
     ///
     /// - Returns: ElementNode representing the specified Paragraph.
     ///
-    func createParagraphNodes(from attrString: NSAttributedString) -> [ElementNode] {
-        guard let paragraphStyle = attrString.attribute(NSParagraphStyleAttributeName, at: 0, effectiveRange: nil) as? ParagraphStyle else {
+    func createParagraphNodes(fromParagraph attrString: NSAttributedString) -> [ElementNode] {
+        guard let style = attrString.attribute(NSParagraphStyleAttributeName, at: 0, effectiveRange: nil) as? ParagraphStyle else {
             return []
         }
 
+        return createParagraphNodes(fromStyle: style)
+    }
+
+
+    /// Extracts the ElementNodes contained within a Paragraph's AttributedString.
+    ///
+    /// - Parameters:
+    ///     - attributes: Paragraph's Attributes from which we intend to extract the ElementNode
+    ///
+    /// - Returns: ElementNode representing the specified Paragraph.
+    ///
+    func createParagraphNodes(fromAttributes attributes: [String: Any]) -> [ElementNode] {
```

Comment:
As commented below, the type should be pretty clear from the variable type, so there should not be a need to specify it in the parameter name.

Can we make this `from attributes`?